### DISCUSSION TITLE
Country/State fields related

### DISCUSSION THREAD

**Discussion by ahmedsliman** [statamic/cms#6717] [Q&A]
Hey team,

I need to ask about a feature I am not sure how to do via Statamic "if possible".

I need a specific Collection to have two fields "countries" and "states" and they are related, which means each state has its own country, How can I make them related in the creation form, "I mean If I choose the country from the select menu the states select menu filtered by this country", is that possible?

Thanks in advance

**Comment by edalzell** [comment]
You'd need to make a custom fieldtype that had two selects in it.
